pkg/kubectl/cmd/create.go

```diff
@@ -97,11 +97,11 @@ func RunCreate(f *cmdutil.Factory, out io.Writer, filenames util.StringList) err
 	err = r.Visit(func(info *resource.Info) error {
 		data, err := info.Mapping.Codec.Encode(info.Object)
 		if err != nil {
-			return err
+			return fmt.Errorf("error when creating %q: %v ", info.Source, err)
```

Comment:
If I am not wrong, this should be an api/error (https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/errors/errors.go#L91).

cc @bgrant0607 @lavalamp